apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/AccessJsonE2E.java

```diff
@@ -54,6 +55,12 @@ public void verifyAccessJson() throws IOException {
         .when()
         .get(requestString);
 
+    // Needed to resolve a race condition
+    try {
+      TimeUnit.MILLISECONDS.sleep(250);
+    } catch (InterruptedException ignored) {
+    }
+
```

Comment:
Wouldn't it be better to use Awaitility for this than just wait an arbitrary amount of time once?  This feels like it will be very brittle.

Reply:
I'll try it out. This was a quick fix reversion of the sleep that was here before; I'm not intimately familiar with what it was trying to avoid. If it was just something like waiting for the access file to be written, awaitility should work.